gui.py

```diff
@@ -188,7 +188,11 @@ def __init__(
     ) -> None:
         self.ui_root = ui_root
 
-        display_window = pygame_gui.elements.UIWindow(
+        self.manager = manager
+
+        self.pressed_keys = set()
+
+        self.display_window = display_window = pygame_gui.elements.UIWindow(
```

Comment:
Two assignments? Is this intended?

Reply:
Nope Removed lol, just needed to go on the object so we can check if it is in the focused set ... this way we only pass kbd input to the VM if the "Display" is the gui element in focus.